packages/ref-data-retrievers/bods-retriever/index.ts

```diff
@@ -30,7 +30,11 @@ export const setupTables = async (dbClient: Kysely<Database>) => {
     }
 };
 
-const getBodsDataAndUploadToS3 = async (bodsUrl: string, txcZippedBucketName: string) => {
+const getBodsDataAndUploadToS3 = async (
+    bodsUrl: string,
+    txcZippedBucketName: string,
+    fileName: string = "bods.zip",
```

Comment:
I'd maybe not bother with the default here and just pass it through explicitly in the function call

Reply:
done